fsck/fsck.c

```diff
@@ -666,7 +666,7 @@ static int exfat_de_iter_get(struct exfat_de_iter *iter,
 			iter->read_size, iter->next_read_offset);
 		if (ret == EOF) {
 			return EOF;
-		} else if (ret <= 0) {
+		} else if (ret < 0) {
```

Comment:
exfat_file_read returns negative, 0, or iter->read_size. So checking that the return value is less or equals to 0 is right. I am removing this function to implement recovery.

Reply:
Ok thanks

Reply:
Any update on this?

Reply:
> Any update on this?

Sorry, I put some comments about commits.